.gitignore

```diff
@@ -7,3 +7,10 @@ npm-debug.log
 
 # WebStorm
 .idea
+
+# Typescript
+*.d.ts
+
+# compiled files
+src/*.js
```

Comment:
If we change the project structure anyway, can we emit the compiled JS and .d.ts in a different folder than `src`? Like `dist`, or `lib`?